docs/resources/vpc_route.md

```diff
@@ -93,6 +95,6 @@ This resource provides the following timeouts configuration options:
 
 VPC routes can be imported using the route table ID and their `destination` separated by a slash, e.g.
 
-```
-$ terraform import flexibleengine_vpc_route.test &ltroute_table_id&gt/&ltdestination&gt
+```shell
+terraform import flexibleengine_vpc_route.test &ltroute_table_id&gt/&ltdestination&gt
```

Comment:
```suggestion
terraform import flexibleengine_vpc_route.test <route_table_id>/<destination>
```

Reply:
It's fixed, I have found 3 files with this typo.